### PR TITLE
Plan pending/approved/suspended/rejected emails

### PR DESCRIPTION
**Pull request by dadambickford**

### CODE REVIEW

File: src/emails/plan-approved.hbs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <p>Hi {{ default.first_name }},</p>
+              <br />
+              <p>We are happy to have you onboard. Your application for a {{ default.plan_name }} IRA has been approved and we are ready to get you started with your retirement&nbsp;plan.</p>
```
Comment (joelcloralt) [inline src/emails/plan-approved.hbs]: any reason why you need a `&nbsp;` here?

File: src/emails/plan-approved.hbs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+              <br />
+              <p>Link your bank account to make your first contribution toward a comfortable&nbsp;retirement.</p>
```
Comment (joelcloralt) [inline src/emails/plan-approved.hbs]: same question about `&nbsp;`

File: src/emails/plan-pending.hbs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: transactional.hbs
-subject: Basic Plan Confirmation
+subject: Plan Pending
```
Comment (joelcloralt) [inline src/emails/plan-pending.hbs]: Update subject to `We're processing your application`

File: src/emails/plan-pending.hbs [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 ---
 <table class="main">
   <tr>
```
Comment (joelcloralt) [inline src/emails/plan-pending.hbs]: We might have forgotten to add the salutation here. `Hi {firstName}`

File: src/emails/plan-rejected.hbs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <p>Hi {{ default.first_name }},</p>
+              <br />
+              <p>We regret to inform you that your application for a {{ default.plan_name }} IRA was not&nbsp;approved.</p>
```
Comment (joelcloralt) [inline src/emails/plan-rejected.hbs]: same question about `&nbsp;`

File: src/emails/plan-suspended.hbs [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+subject: Plan Suspended
+---
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <h1>Uh Oh!</h1>
```
Comment (joelcloralt) [inline src/emails/plan-suspended.hbs]: At the end Cvo said to remove the Uh oh.

Comment (dadambickford) [inline src/emails/plan-suspended.hbs]: removed.

File: src/emails/plan-pending.hbs (hunk 4 above)
Comment (dadambickford) [inline src/emails/plan-pending.hbs]: looks like it was just hidden in the fold, it's old so it wasn't in the PR changes

**Comment by dadambickford** [comment]
requested changes changed @joelcloralt 👍

File: src/emails/plan-approved.hbs [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+subject: Your application has been approved!
+---
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <p>Hi {{ default.first_name }},</p>
```
Comment (joelcloralt) [inline src/emails/plan-approved.hbs]: Add comma after Hi and exclamation mark after name.

File: src/emails/plan-rejected.hbs [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+subject: Important message regarding your application
+---
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <p>Hi {{ default.first_name }},</p>
```
Comment (joelcloralt) [inline src/emails/plan-rejected.hbs]: Add comma after Hi and exclamation mark after name.

File: src/emails/plan-suspended.hbs [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+subject: Additional information is required
+---
+<table class="main">
+  <tr>
+    <td class="wrapper align-center">
+      <table>
+        <tr>
+          <td>
+            <center>
+              <p>Hi {{ default.first_name }},</p>
```
Comment (joelcloralt) [inline src/emails/plan-suspended.hbs]: Add comma after Hi and exclamation mark after name.

**Pull request merged**